src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ClientTrackingConfiguration.kt

```diff
@@ -27,24 +23,35 @@ class ClientTrackingConfiguration(private val clientTrackingInterceptor: ClientT
 }
 
 @Configuration
-class ClientTrackingInterceptor() : HandlerInterceptor {
+class ClientTrackingInterceptor : HandlerInterceptor {
+
+  companion object {
+    private val LOG = LoggerFactory.getLogger(this::class.java)
```

Comment:
Slightly unusual syntax 😄

Reply:
agreed but this is what we use in vsip :)